exchanges/huobi/huobi_wrapper.go

```diff
@@ -1683,13 +1614,13 @@ func (h *HUOBI) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end
 	if err := h.ValidateKline(pair, a, interval); err != nil {
 		return kline.Item{}, err
 	}
-	formattedPair, err := h.FormatExchangeCurrency(pair, a)
+	symbolValue, err := h.FormatSymbol(pair, asset.Spot)
 	if err != nil {
 		return kline.Item{}, err
 	}
 	klineParams := KlinesRequestParams{
 		Period: h.FormatExchangeKlineInterval(interval),
-		Symbol: formattedPair.String(),
+		Symbol: symbolValue,
```

Comment:
change symbol type to currency.Pair and do the formatting in the GetSpotKline func